README.md

```diff
@@ -179,9 +179,14 @@ Support for multiple RDBMS is a work in progress.
 
 ### SQL Editor
 
-| Key      | Action                |
-| -------- | --------------------- |
-| CTRL + R | Run the SQL statement |
+| Key          | Action                            |
+| ------------ | --------------------------------- |
+| CTRL + R     | Run the SQL statement             |
+| CTRL + Space | Open external editor(Linux only)  |
```

Comment:
```suggestion
| CTRL + Space | Open external editor (Linux only)  |
```

What about macOS?

I would say the problem is
Windows

Reply:
I've added this feature only for linux and it shouldn't do anything on any other os.